SPECS/atftp/atftp.signatures.json

```
@@ -1,5 +1,5 @@
 {
  "Signatures": {
-  "atftp-0.7.5.tar.gz": "93c87a4fb18218414e008e01c995dadd231ba4c752d0f894b34416d1e6d3038a"
+  "atftp-0.8.0.tar.gz": "df2aa089c7670f9eab40e5598e5d2cb6a582dc5182926ea50b4d690e4e37f316"
  }
 }
```

nit - end of file newline
```suggestion
}

```